doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -381,35 +367,41 @@ Modem libraries
-      When such an occurrence is detected, GNSS is stopped without waiting for a fix or a timeout.
+    * Added:
+
+      * Support for P-GPS data retrieval from an external source, implemented separately by the application.
+        Enabled by setting the :kconfig:option:`CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL` option.
+        The library triggers a :c:enum:`LOCATION_EVT_GNSS_PREDICTION_REQUEST` event when assistance is needed.
+      * A new feature, obstructed satellite visibility detection for GNSS.
+        When this feature is enabled, the library tries to detect occurrences where getting a GNSS fix is unlikely or would consume a lot of energy.
+        When such an occurrence is detected, GNSS is stopped without waiting for a fix or a timeout.
+
+    * Updated:
+
+      * The :c:member:`request` member of the :c:struct:`location_event_data` structure renamed to :c:member:`agps_request`.
+      * Current system time is attached to the ``location_datetime`` parameter of the location request response with Wi-Fi and cellular methods.
+        The timestamp comes from the moment of scanning or neighbor measurements.
+      * Removed dependency on the :ref:`lib_modem_jwt` library.
+        The :ref:`lib_location` library now selects :kconfig:option:`CONFIG_NRF_CLOUD_REST_AUTOGEN_JWT` when using :kconfig:option:`CONFIG_NRF_CLOUD_REST`.
 
   * :ref:`nrf_modem_lib_readme` library:
 
-    * Added :c:macro:`NRF_MODEM_LIB_ON_INIT` macro for compile-time registration of callbacks on modem initialization.
-    * Added :c:macro:`NRF_MODEM_LIB_ON_SHUTDOWN` macro for compile-time registration of callbacks on modem de-initialization.
+    * Added:
+
+      * :c:macro:`NRF_MODEM_LIB_ON_INIT` macro for compile-time registration of callbacks on modem initialization.
+      * :c:macro:`NRF_MODEM_LIB_ON_SHUTDOWN` macro for compile-time registration of callbacks on modem de-initialization.
+      * :kconfig:option:`CONFIG_NRF_MODEM_LIB_LOG_FW_VERSION_UUID` to enable logging for both FW version and UUID at the end of the library initialization step.
+
     * Deprecated :c:func:`nrf_modem_lib_shutdown_wait` function, in favor of :c:macro:`NRF_MODEM_LIB_ON_INIT`.
-    * Added :kconfig:option:`CONFIG_NRF_MODEM_LIB_LOG_FW_VERSION_UUID` to enable logging for both FW version and UUID at the end of the library initialization step.
 
   * :ref:`lte_lc_readme` library:
 
-    * Added :c:macro:`LTE_LC_ON_CFUN` macro for compile-time registration of callbacks on modem functional mode changes using :c:func:`lte_lc_func_mode_set`.
-    * Added support for simple shell commands.
-
-* Removed:
+    * Added:
 
-  * A-GPS library:
+      * :c:macro:`LTE_LC_ON_CFUN` macro for compile-time registration of callbacks on modem functional mode changes using :c:func:`lte_lc_func_mode_set`.
+      * Support for simple shell commands.
 
-    * Removed the deprecated A-GPS library.
+* Removed the deprecated A-GPS library.
 
 Libraries for networking
 ------------------------
```

Comment:
Edit to line 416: I think "Changes to" can be cut from that line, as "Updated changes to" sounds weird.

Reply:
Fixed